custom/enikshay/private_sector_datamigration/management/commands/create_cases_by_beneficiary.py

```diff
@@ -92,24 +101,73 @@ def handle(self, domain, **options):
         else:
             location_owner = None
 
-        # Assert never null
-        assert not beneficiaries.filter(firstName__isnull=True).exists()
-        assert not beneficiaries.filter(lastName__isnull=True).exists()
-        assert not beneficiaries.filter(phoneNumber__isnull=True).exists()
-        assert not Episode.objects.filter(dateOfDiagnosis__isnull=True).exists()
-        assert not Episode.objects.filter(patientWeight__isnull=True).exists()
-        assert not Episode.objects.filter(rxStartDate__isnull=True).exists()
-        assert not Episode.objects.filter(site__isnull=True).exists()
-        assert not Adherence.objects.filter(creationDate__isnull=True).exists()
+        assert not case_ids or not owner_state_id
+        assert owner_state_id or not owner_district_id
```

Comment:
Maybe not too important here, but for best practice...

Assertions should be used to verify internal consistency of program logic, and it should not be possible to for a user to make them fail by manipulating program input (an assertion failure should always indicate a bug). I think these should be changed to `if` statements that raise `CommandError`, ideally with helpful error messages, since they appear to be validating user input.